app/components/related_works/edit_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<div data-controller="related-work" data-related-work-citation-container-selector-value=".field-container">
+  <%= render Elements::Forms::TextFieldComponent.new(form:, field_name: :identifier, label: helpers.t('related_works.edit.fields.identifier.label.html'), disabled: citation_present?,
+                                                     data: { related_work_target: 'identifier' }) %>
+  <%= render Elements::Forms::CheckboxComponent.new(form:, field_name: :use_citation, label: helpers.t('related_works.edit.fields.use_citation.label'),
+                                                    data: { action: 'related-work#toggleCitation' }, checked: citation_present?) %>
```

Comment:
Should use_citation be initialized to citation_present? so that checked does not need to be explicitly set for checkbox?

Reply:
I made a change related to this feedback. Check it out?